cypress/support/commands.js

```diff
@@ -23,3 +25,6 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+Cypress.Commands.add('findById', (id) => {
+  return cy.get(`#${id}`)
+})
```

Comment:
The `findById` command is redundant because it simply wraps `cy.get` with an ID selector. Consider using `cy.get('#id')` directly in your tests for simplicity and to avoid unnecessary abstraction.